File: P3_app_durcevic/app/src/main/java/ffos/p3/ontologija/Ontologija.java
package ffos.p3.ontologija;

import java.io.Serializable;

public class Ontologija implements Serializable {

    public Ontologija() {
        this.sifra = sifra;
        this.naziv = naziv;
        this.izvodac = izvodac;
        this.rockZanr = rockZanr;
        this.drzava = drzava;
        //this.godina = godina;
        this.brojPjesama = brojPjesama;

    }

    private int sifra;
    private String naziv;
    private String izvodac;
    private String rockZanr;
    private String drzava;
    //private String godina;
    private String brojPjesama;

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public String getNaziv() {
        return naziv;
    }

    public void setNaziv(String naziv) {
        this.naziv = naziv;
    }

    public String getIzvodac() {
        return izvodac;
    }

    public void setIzvodac(String izvodac) {
        this.izvodac = izvodac;
    }

    public String getRockZanr() {
        return rockZanr;
    }

    public void setRockZanr(String rockZanr) {
        this.rockZanr = rockZanr;
    }

    public String getDrzava() {
        return drzava;
    }

    public void setDrzava(String drzava) {
        this.drzava = drzava;
    }

    public String getBrojPjesama() {
        return brojPjesama;
    }

    public void setBrojPjesama(String brojPjesama) {
        this.brojPjesama = brojPjesama;
    }
}

File: P3_ontologija_durcevic/src/Durcevic/Ontologija.php
<?php

namespace Durcevic;

/**
 * @Entity @Table(name="ontologija")
 **/


class Ontologija 

{
    /** @id @Column(type="integer") @GeneratedValue **/
    protected $sifra;

    /**
    * @Column(type="string")
    */
    private $naziv;

     /**
    * @Column(type="string")
    */
    private $izvodac;

     /**
    * @Column(type="string")
    */
    private $rockZanr;

     /**
    * @Column(type="string")
    */
    private $drzava;

     /**
    * @Column(type="integer")
    */
    //private $godina;

    /**
    * @Column(type="string")
    */
    private $brojPjesama;

    public function getSifra(){
      return $this->sifra;
    }
  
    public function setSifra($sifra){
      $this->sifra = $sifra;
    }
  
    public function getNaziv(){
      return $this->naziv;
    }
  
    public function setNaziv($naziv){
      $this->naziv = $naziv;
    }
  
    public function getIzvodac(){
      return $this->izvodac;
    }
  
    public function setIzvodac($izvodac){
      $this->izvodac = $izvodac;
    }
  
    public function getRockZanr(){
      return $this->rockZanr;
    }
  
    public function setRockZanr($rockZanr){
      $this->rockZanr = $rockZanr;
    }
  
    public function getDrzava(){
      return $this->drzava;
    }
  
    public function setDrzava($drzava){
      $this->drzava = $drzava;
    }
  
    //public function getGodina(){
      //return $this->godina;
    //}
  
    //public function setGodina($godina){
      //$this->godina = $godina;
    //}
  
    public function getBrojPjesama(){
      return $this->brojPjesama;
    }
  
    public function setBrojPjesama($brojPjesama){
      $this->brojPjesama = $brojPjesama;
    }
    public function setPodaci($podaci)
    {
      foreach($podaci as $kljuc => $vrijednost){
        $this->{$kljuc} = $vrijednost;
      }
    }

}

?>